cypress/e2e/hermioneAccount.spec.cy.js

```
@@ -0,0 +1,89 @@
+/// <reference types='cypress' />
+
+import { faker } from '@faker-js/faker';
+
+describe('Bank app', () => {
+  const user = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  let initialBalance = 5096;
+
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.contains('.btn', 'Customer Login').click();
+    cy.get('[name="userSelect"]').select(user);
+    cy.contains('.btn', 'Login').click();
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', initialBalance)
```

This selector is brittle because `.contains('strong', initialBalance)` relies on the balance being exactly the hardcoded value (`5096`). If the balance changes from a previous test run, this selector will fail. A more robust approach would be to select the balance element based on its position (e.g., the second `<strong>` element), get its text, and then store it, without assuming its initial value.